apps/autoscheduler/Derivative.cpp

```diff
@@ -5,7 +5,6 @@
 #include <set>
 
 #include "Halide.h"
-#include "Errors.h"
```

Comment:
Errors.h is supposed to be apps/autoscheduler/Errors.h - there's no Errors.h in Halide proper.

Reply:
Well darn. I misread, that one little "s" caught me. Will fix and update this PR as appropriate.